core/crypto/sha/sha256.cpp

```diff
@@ -0,0 +1,13 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "crypto/sha/sha256.hpp"
+
+namespace fc::crypto::sha {
+
+  libp2p::common::Hash256 sha256(gsl::span<const uint8_t> input) {
+    return libp2p::crypto::sha256(input).value();
```

Comment:
remove `sha` cmake target or make it `INTERFACE`.  
move implementation to header.

Reply:
I prefer to keep this file. So if signature or library is changed, we can update with a single file.
Moreover, `sha256(x)` is more clean than `sha256(x).value()`